plugins/woocommerce/client/legacy/js/frontend/cart.js

```diff
@@ -761,6 +766,20 @@ jQuery( function ( $ ) {
 			} );
 		},
 
+		/**
+		 * Handle when pressing the Space key on the remove item link.
+		 * This is necessary because the link got the role="button" attribute
+		 * and needs to act like a button.
+		 *
+		 * @param {Object} evt The JQuery event
+		 */
+		on_keydown_remove_item: function ( evt ) {
+			if( evt.key === ' ' ) {
+				evt.preventDefault();
+				$( evt.currentTarget ).trigger( 'click' );
+			}
+		},
```

Comment:
Similar as before, changing `evt` to `event` would increase the readability here:

```suggestion
		on_keydown_remove_item: function ( event ) {
			if ( event.key === ' ' ) {
				event.preventDefault();
				$( event.currentTarget ).trigger( 'click' );
			}
		},
```